requirements-ftest.txt

```diff
@@ -1,6 +1,6 @@
-avocado-framework==108.0
-avocado-framework-plugin-result-html==108.0
-avocado-framework-plugin-varianter-yaml-to-mux==108.0
+avocado-framework==82
+avocado-framework-plugin-result-html==82
+avocado-framework-plugin-varianter-yaml-to-mux==82
```

Comment:
Doesn't this just make Dependabot want to push a PR to update this to `108.0` the next time it runs, and every time thereafter when we add commits to it's PRs to back out the updates?

Interesting that it found a file named `requirements-ftest.txt` though.

Reply:
Yeah, I think dependbot will push another PR to update this again, which we don't have to merge. I'd like to see what it does though, and we can add an "exclude" to the config for this if needed